src/common/legacy_config_opts.h

```diff
@@ -1543,3 +1543,7 @@ OPTION(rgw_sts_entry, OPT_STR)
 OPTION(rgw_sts_key, OPT_STR)
 OPTION(rgw_s3_auth_use_sts, OPT_BOOL)  // should we try to use sts for s3?
 OPTION(rgw_sts_max_session_duration, OPT_U64) // Max duration in seconds for which the session token is valid.
+OPTION(rgw_sts_token_introspection_url, OPT_STR)  // url for introspecting web tokens
```

Comment:
what does introspection allow us to do?

Reply:
An OAuth2.0 Access token is by definition opaque to the client. Token introspection in OAuth2.0, validates a token with the IDP that issued it. The IDP sends back a  JSON response, which tells us whether it is valid or not and other metadata. https://tools.ietf.org/html/rfc7662